services/service-tester.js

```diff
@@ -92,14 +90,16 @@ class ServiceTester {
   /**
    * Register the tests with Mocha.
    */
-  toss() {
-    const specs = this.specs
+  toss({ baseUrl, skipIntercepted }) {
```

Comment:
Inject config. This happens in `toss()` because that gets invoked by the tester, whereas the constructor is invoked directly by the tester modules.